### PR TITLE
Remove dead comments from gradle files

### PR DESCRIPTION
**Pull request by 8Keep**
Deleting some dead code here to simplify the gradle files.

### CODE REVIEW

File: jme3-examples/build.gradle [hunk 1]
```diff
@@ -20,12 +20,9 @@ dependencies {
     implementation project(':jme3-jbullet')
     implementation project(':jme3-jogg')
     implementation project(':jme3-lwjgl')
-//    implementation project(':jme3-lwjgl3')
```
Comment (stephengold) [inline jme3-examples/build.gradle:23]: This commented-out line makes it easier for people to switch between LWJGL v2 and v3, which is something I used to do a lot during beta testing of releases. With a little scripting, that functionality could be moved the gradle.properties file, which might make it easier to find.

File: jme3-examples/build.gradle [hunk 2]
```diff
@@ -20,12 +20,9 @@ dependencies {
...
     implementation project(':jme3-networking')
     implementation project(':jme3-niftygui')
     implementation project(':jme3-plugins')
-//    implementation project(':jme3-plugins-json')
```
Comment (stephengold) [inline jme3-examples/build.gradle:27]: I commented these out as part of PR #2278. I'm hoping they can be put back in after somebody figures out why the jme3-plugins JSON implementation wasn't working.

**Comment by 8Keep** [comment]
Reverted changes to jme3-examples build file

**Comment by stephengold** [comment]
Thank you, @8Keep.
The rest of the dead comments are mysteries to me.
Have you looked at the commits that added them, to understand why they're there?

**Comment by 8Keep** [comment]
The ones I left in this commit were last changed 10+ years ago, I am pretty sure they were just left by the authors

**Comment by stephengold** [comment]
You're probably right! Thanks.

**Pull request merged**